app/src/ext_power_generic.c

```diff
@@ -210,7 +215,8 @@ static int ext_power_generic_pm_control(const struct device *dev, uint32_t ctrl_
 static const struct ext_power_generic_config config = {
     .label = DT_INST_GPIO_LABEL(0, control_gpios),
     .pin = DT_INST_GPIO_PIN(0, control_gpios),
-    .flags = DT_INST_GPIO_FLAGS(0, control_gpios)};
+    .flags = DT_INST_GPIO_FLAGS(0, control_gpios),
+    COND_CODE_0(DT_INST_NODE_HAS_PROP(0, init_delay_ms), (0), (DT_INST_PROP(0, init_delay_ms)))};
```

Comment:
```suggestion
    .init_delay_ms = DT_INST_PROP_OR(0, init_delay_ms, 0)};
```

Use `DT_INST_PROP_OR`. I know there's other code that uses this approach, but it was before I found that helper.